README.md

```diff
@@ -1,23 +1,11 @@
 # q1k_eeget_init
 Initialization code for the EEG/ET data from the Q1K project.
 
-# setup
-
-Assumes that https://github.com/jadesjardins/mne-python.git is a fork of git@github.com:scott-huberty/mne-python.git
-
 ## clone stuff
 ```bash
 git clone https://github.com/lina-usc/q1k_eeget_init.git
-cd q1k_eeget_init
-git clone https://github.com/jadesjardins/mne-python.git
-```
```

Comment:
changed this to install mne-tools/mne-python, not anyone elses fork. Also changed this so that the git clone of `mne` is _not_  done inside the `q1k_eeg_et_init` folder.